spec/acceptance/has_interface_with_spec.rb

```diff
@@ -1,52 +0,0 @@
-require 'spec_helper_acceptance'
```

Comment:
where did the acceptance tests go  ?

Reply:
The acceptance test duplicated functionality tested in unit. Do you think this is worth keeping as an acceptance test?